docs/usecases/external_collaboration.rst

```diff
@@ -0,0 +1,364 @@
+.. _usecase_external_collaboration:
+
+Collaborations on sensitive data with remote analyses
+-----------------------------------------------------
+
+This usecase is based on a common problem:
+An external collaborator would like to perform an analysis on a dataset that can't be shared outside of the lab.
+With DataLad, the collaboration can succeed:
+
+#. A mock dataset with the same structure (organization, file names, potentially mock-content) but without (access to) the actual sensitive data is shared or made publicly available instead of the original dataset.
+#. This mock dataset can assist others to develop their code, without disclosing personal information.
+#. External collaborators create a new dataset for their analysis code, with the mock dataset as a subdataset.
+#. The collaborator submits their dataset back to the lab that has all of the data. After code review, the code is executed on the actual data, its results are captured in the submitted dataset, and only results are pushed back to the external collaborators.
+
+The Challenge
+^^^^^^^^^^^^^
+
+The ABC-Lab collected data from patients for one of their studies.
+After publication, a Post-doc from the XYZ-Lab reaches out and proposes an interesting analysis of the same data on a new research question.
+Everyone involved is excited about this collaboration:
+The XYZ-Lab is happy that such a valuable dataset exists and is eager to give credit and co-authorship to collaborators in the ABC-Lab.
+The ABC-Lab is happy that someone else is interested in using their data and appreciates the possibility of another publication with lab-members as co-authors.
+
+However, the legal department in ABC's institute steps in: The data can not be shared with anyone outside as this would be a privacy violation.
+The data contains potentially identifying or personal information of the patients it was collected from.
+
+With such legal problems, the promising collaboration ceases before it even started, the data remains disclosed but also unused, and an interesting research question remains unanswered.
```

Comment:
I stumbled upon the phrase "disclosed but also unused". I am not sure what "disclosed" wants to communicate here? Maybe just

```suggestion
With such legal problems, the promising collaboration ceases before it even started, the data remains unused, and an interesting research question unanswered.
```